code/obj/item/instruments.dm

```diff
@@ -658,6 +659,23 @@ TYPEINFO(/obj/item/instrument/bikehorn/dramatic)
 				sleep(5 SECONDS)
 		return 1
 
+/obj/item/instrument/whistle/security
+	name = "security whistle"
+	desc = "A whistle. Good for getting attention. This one has a red stripe on it."
+	icon_state = "whistle-sec"
+	contraband = 4 //beepsky takes stolen whistles seriously
+	HELP_MESSAGE_OVERRIDE("Blow this to briefly command nearby securitrons to follow your pointing, point at a perp to have them arrested.")
+
+	post_play_effect(mob/user)
+		var/list/bots = list()
+		for (var/obj/machinery/bot/secbot/secbot in view(user.client.view, user)) //cursed awful byond screen syntax
```

Comment:
why not hearers() for audio cue?

Reply:
`hearers` only returns mobs, first thing I tried!

Reply:
mobcritter securitron when :(